Cargo.toml

```diff
@@ -84,45 +84,45 @@ toml = "0.8.11"
 tracing = "0.1.40"
 typenum = "1.17.0"
 libp2p = { package = "libp2p", version = "0.53.2", features = [
-        "macros",
-        "autonat",
-        "cbor",
-        "dns",
-        "floodsub",
-        "gossipsub",
-        "identify",
-        "kad",
-        "mdns",
-        "noise",
-        "ping",
-        "plaintext",
-        "pnet",
-        "quic",
-        "relay",
-        "request-response",
-        "rendezvous",
-        "secp256k1",
-        "serde",
-        "tcp",
-        "uds",
-        "websocket",
-        "yamux",
+  "macros",
```

Comment:
What did you run to format this? Is it a vscode command?

Reply:
Actually been trying out neovim and it was doing the toml formatting

Reply:
Oooh I might bug you for your config later